content/docs/reactivesearch/vue/list/MultiDropdownList.md

```diff
@@ -192,6 +192,35 @@ You can use render as a slot as shown below:
 </template>
 ```
 
+-   **renderNoResults** `String|Function|slot-scope` [optional]
+    show custom message or component when no results found.
+
+<!-- prettier-ignore -->
+```html
+<template
+    slot="renderNoResults"
+>
+	<h4>No Results Found!</h4>
+</template>
+
+<!-- or -->
+
+<MultiDropdownList
```

Comment:
@anik-ghosh-au7 Can you use kebab-case for components name?
Make that change at all places.
```suggestion
<multi-dropdown-list
```